addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -16,22 +21,143 @@ Chatter.components = {
     FollowerList,
 };
 
-Chatter.props = [...Chatter.props, "has_activities?"];
+Chatter.props = [...Chatter.props, "has_activities?", "hasParentReloadOnFollowersUpdate?"];
```

Comment:
Shouldn't it be a push?

Reply:
Both syntax works. If there is a prefer, I can change it.